clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -545,6 +546,12 @@ public static void main(String args[]) throws Exception {
         case CONFIGURE_STORE_VIEW:
           configureStoreView(cmd);
           break;
+        case BACKUP_STORE_METADATA_FROM_GRAVEYARD:
```

Comment:
Might be good to add an example of how to trigger this new command as well in the commit message.

Reply:
Sure